tests/integration/test_text_classification.py

```diff
@@ -7,25 +7,27 @@
 import pytest
 import torch
 
-from biome.text import Pipeline, TrainerConfiguration, VocabularyConfiguration
+from biome.text import Pipeline, Dataset, TrainerConfiguration, VocabularyConfiguration
 from biome.text.configuration import CharFeatures
 from biome.text.configuration import WordFeatures
-from biome.text.data import DataSource
 
 
 @pytest.fixture
-def train_valid_data_source() -> Tuple[DataSource, DataSource]:
+def train_valid_data_source() -> Tuple[Dataset, Dataset]:
```

Comment:
For consistency reason i would rename this fixture to `train_valid_dataset`
```suggestion
def train_valid_dataset() -> Tuple[Dataset, Dataset]:
```

Reply:
yes, i totally forgot to do that myself